js/battle.js

```diff
@@ -2378,6 +2378,7 @@ var Battle = (function () {
 			else if (this.gen <= 4) this.backdropImage = 'fx/' + BattleBackdropsFour[Math.floor(Math.random() * BattleBackdropsFour.length)];
 			else if (this.gen <= 5) this.backdropImage = 'fx/' + BattleBackdropsFive[Math.floor(Math.random() * BattleBackdropsFive.length)];
 		}
+		if (this.gen <=6 && Tools.prefs('bwgfx')) this.backdropImage = 'fx/' + BattleBackdropsFive[Math.floor(Math.random() * BattleBackdropsFive.length)];
```

Comment:
Does `this.gen <= 6` even need to be included? It will always be true anyway (at least until gen 7).